x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -60,132 +113,227 @@ const WatchListUi = ({ intl }: { intl: InjectedIntl }) => {
     );
   }
 
-  const columns = [
-    {
-      field: 'id',
-      name: i18n.translate('xpack.watcher.sections.watchList.watchTable.idHeader', {
-        defaultMessage: 'ID',
-      }),
-      sortable: true,
-      truncateText: true,
-      render: (id: string) => {
-        return (
-          <EuiLink
-            className="indTable__link euiTableCellContent"
-            data-test-subj={`watchIdColumn-${id}`}
-            href={`#/management/elasticsearch/watcher/watches/watch/${id}/status`}
-          >
-            {id}
-          </EuiLink>
-        );
+  let content;
+
+  if (availableWatches && availableWatches.length === 0) {
+    content = (
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        title={
+          <h1>
+            <FormattedMessage
+              id="xpack.watcher.sections.watchList.emptyPromptTitle"
+              defaultMessage="You don't have any watches yet"
+            />
+          </h1>
+        }
+        body={
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.watcher.sections.watchList.emptyPromptDescription"
+                defaultMessage="Start by creating a watch."
+              />
+            </p>
+          </Fragment>
+        }
+        actions={createWatchButtons}
+      />
+    );
+  } else {
+    const columns = [
+      {
+        field: 'id',
+        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.idHeader', {
+          defaultMessage: 'ID',
+        }),
+        sortable: true,
+        truncateText: true,
+        render: (id: string) => {
+          return (
+            <EuiLink
+              className="indTable__link euiTableCellContent"
```

Comment:
Is this class name left over from a copy/paste? `indTable__link`

Reply:
yep, looks that way. deleted the class.